coding-guides/README.md

```diff
@@ -1,4 +1 @@
 # Coding Guides
-
```

Comment:
if I'm right this is going to complain that there is a missing newline

Reply:
My bad, I'll patch up

Reply:
It was committed with a newline at the end, I have `files.insertFinalNewLine` set to true. Not sure why that happened though!

Reply:
I'll just resolve this, because I really don't know what is happening with this.